packages/autotask-client/src/api.ts

```diff
@@ -77,6 +78,24 @@ export class AutotaskClient extends BaseApiClient {
     return this.updateCode(autotaskId, encodedZippedCode);
   }
 
+  public async listAutotaskRuns(autotaskId: string): Promise<AutotaskRunListResponse> {
```

Comment:
@pasevin 
PR looks great. Thanks!

Minor thing is that we are not handling pagination via cli call. List endpoint returns 20 results by default. Next cursor is available in the response.

We should extend this function to receive optional next param and forward it as part of the url(query param).

Reply:
good point, totally skipped my mind!

Reply:
Hope [this is](https://github.com/OpenZeppelin/defender-client/pull/99/commits/58e8770c0b96578dc4fd81a308cbedb73d12a511) what you have in mind 🙂

Reply:
Yes, that's what I had in mind. Thanks for adding it.